Sources/Smile.swift

```diff
@@ -233,3 +295,31 @@ fileprivate extension String {
     return components(separatedBy: .decimalDigits).joined()
   }
 }
+
+// MARK: - emojiRules
+
+/// Run supplied emojiRule against string
+@available(iOS 10.2, *)
+public func checkEmojiRule(emojiRule: emojiRule, string: String) -> Any{
```

Comment:
What do you think of omitting this function? I guess users can use the methods isEmojiOnly, countNonEmoji directly? This method returns Any and introduces another enum `emojiRule

Reply:
Agreed. I was using this function to assist in testing the rules easier. Not sure why I decided to leave it in there.